packages/next/next-server/lib/utils.ts

```diff
@@ -191,6 +191,12 @@ export interface NextApiRequest extends IncomingMessage {
   body: any
 
   env: Env
+
+  preview: true | undefined
```

Comment:
```suggestion
  preview?: boolean
```

To match `GetStaticPropsContext` & `GetServerSidePropsContext`.

Reply:
It makes sense to me for the compiler to warn on `preview === false`, as this will always evaluate to false.
Should this be `preview?: true` (and `GetStaticPropsContext`/`GetServerSidePropsContext` updated for consistency)?
```suggestion
  preview?: true
```